src/main/java/carRace/step5/domain/Car.java

```diff
@@ -0,0 +1,41 @@
+package carRace.step5.domain;
+
+
+import carRace.step5.domain.util.CarNamesCheck;
+
+public class Car {
+    private String carName;
+    private int moveDistance;
+    private static int referenceValue = 4;
```

Comment:
클래스 작성시, 상수 다음 인스턴스 변수가 오는게 관례이오니 참고 부탁드립니다 :)
```suggestion
    private static int referenceValue = 4;
    
    private String carName;
    private int moveDistance;
```